projects/01-card-validation/README.pt-BR.md

```diff
@@ -0,0 +1,280 @@
+# Cartão de Crédito Válido
+
+## Índice
+
+* [1. Preâmbulo](#1-preâmbulo)
```

Comment:
"Preâmbulo" é um termo que a gente não usa mt no cotidiano né? Podia trocar pra "introdução", "Início" ou qualquer coisa do tipo, o que acha?

Reply:
Introdução fica bom.